addon/components/em-select.js

```diff
@@ -10,7 +10,7 @@ Syntax:
     optionValuePath=keyForValue
     optionLabelPath=keyForLabel
     prompt="Optional default prompt"}}
-
+    optionDisabledPath=keyForDisabled
```

Comment:
Your new argument is outside the end of the tag `}}`.

You also missed to define it as null on the body of FormGroupComponent.